src/test/java/com/mongodb/jdbc/MongoDriverTest.java

```diff
@@ -93,18 +93,12 @@ void testBasicURL() throws SQLException {
     }
 
     @Test
-    void testDBURL() throws SQLException {
+    void testURIDBURL() throws SQLException {
```

Comment:
Can you add a test which shows what happens when their is a default DB supplied in both the URI and the properties.
- For `jdbc:mongodb://localhost/foo?authSource=admin` + `database=myPropsDB`, the result should be default DB = myPropsDB, authSource = admin.
- For `jdbc:mongodb://localhost/admin` + `database=foo`, the result should be default DB = foo, authSource = admin.